openage/log/logging.py

```diff
@@ -63,9 +63,8 @@ def log(lvl, msg, stackframes=1):
         # the C++ interface is uninitialized.
         with LOCK:  # pylint: disable=not-context-manager
             if lvl.value >= Level.current.value:
-                print("\x1b[" + level_colorcode(lvl) + "m" +
-                      lvl.name.upper().rjust(4) + "\x1b[m " +
-                      msg)
+                print(''.join(["\x1b[", level_colorcode(lvl), "m",
+                               lvl.name.upper().rjust(4), "\x1b[m ", msg]))
```

Comment:
you don't have to join the string, print can have multiple arguments that will be printed sequentially